app-kotlin/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="ly.count.android.demo.kotlin">
 
-  <uses-permission android:name="android.permission.INTERNET" />
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.INTERNET"/>
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+  <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>

```

Comment:
this new? necessary?

Reply:
acutally this is not necessary. While I was building for 35 it gave error that this is missing permissions. But not now reverting it